src/components/BetaWarning.js

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
-import { Menu, Responsive } from 'semantic-ui-react';
+import {
+  Menu,
+  Responsive,
+} from 'semantic-ui-react';
+
 
 const BetaWarning = function (props) {
   return (
     <Menu.Item>
       <Responsive
-        as='strong'
+        as = { `strong` }
         { ...Responsive.onlyTablet }>
         This tool is a prototype.
       </Responsive>
       <Responsive
-        as='strong'
+        as = { `strong` }
         { ...Responsive.onlyComputer }>
         This tool is a prototype and should not be used to make financial decisions.
       </Responsive>
     </Menu.Item>
   );
-};  // End BetaWarning(<>)
+};
```

Comment:
Why remove the end comment? Is it because the component definition is short enough to fit on a screen? I kind of feel like if we want to use these end comments, we should use them consistently.

Reply:
Well, if there were a three line function I don't imagine using and `// Ends...` comment, so the question is really where do we draw the line. Any ideas?

Reply:
Or maybe we use them at the end of every function, if statement, and for loop. That's also an option.

Reply:
Thoughts?